desktops/kde.txt

```diff
@@ -0,0 +1,434 @@
+|                                                                              |
++------------------------------------------------------------------------------+
+
+Finally, 
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ exit                                                                     |
+|   $ reboot                                                                   |
+|   $ startx                                                                   |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+You should now be greeted with a fresh Plasma Desktop!
+
+
+[3.0] Starting KDE
+________________________________________________________________________________
+
+You have two choices on how to launch KDE. Either you can start it directly from
+the console, or you can make use of a login manager. 
+
+
+    [3.1] Console
+    ____________________________________________________________________________
+
+    Launching a KDE session directly from the console is very straightforward.
+    If you would like to use wayland instead of X, replace x11 by wayland in the
+    below command.
+    Simply add a line to your xinitrc file:
+
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   $ launch="exec dbus-launch --exit-with-session startplasma-x11"        |
+    |   $ echo "$launch" >> "$HOME/.xinitrc"                                   |
+    |                                                                          |
+    +--------------------------------------------------------------------------+
+
+
+    [3.2] Login Manager
+    ____________________________________________________________________________
+
+    The KISS-kde repository also includes sddm [7], the Simple Desktop Display
+    Manager, as an option for a login manager. Enabling it is very easy and it
+    comes preconfigured with the breeze theme. First, build and install sddm.
+
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   $ kiss b sddm && kiss i sddm                                           |
+    |                                                                          |
+    +--------------------------------------------------------------------------+
+
+    Then, enable the relevant services. polkit and elogind should already be
+    installed if you followed the prior instructions. If not, install them in
+    the usual way.
+    
+    NOTE: by default, it may not be possible for root to login via sddm. Best
+    practices would dictate that you login as a nonroot user.
```

Comment:
`nonroot` -> `non-root` (or maybe "normal"?)

Reply:
My background is showing. 
Does 'unprivileged' sound better?

Reply:
Yes!